na-toolbar.html

```diff
@@ -49,8 +49,12 @@
     </style>
     <iron-icon icon="bookmark"></iron-icon>
     <h1>Notes</h1>
-    <iron-icon icon="cloud" hidden$="[[!online]]"></iron-icon>
-    <iron-icon icon="cloud-off" hidden$="[[online]]"></iron-icon>
+    <span hidden$="[[!online]]">
+      <iron-icon icon="cloud"></iron-icon>
+    </span>
+    <span hidden$="[[online]]">
+      <iron-icon icon="cloud-off"></iron-icon>
+    </span>
```

Comment:
@bicknellr nope it is not an older version. It seems like a bug:

![image](https://cloud.githubusercontent.com/assets/5981958/24943146/f02a09ac-1f09-11e7-86df-4122e5e50257.png)

the hidden is being overriden by the following style:
![image](https://cloud.githubusercontent.com/assets/5981958/24943191/30fe21fc-1f0a-11e7-98ab-a789966c06c1.png)

https://github.com/PolymerElements/iron-icon/pull/80/files#diff-cd2db018cf5426f07b02bca3f3a1d41bR89